test/tasks/mock/template/SetGameTypeTemplate.sol

```diff
@@ -67,7 +68,7 @@ contract SetGameTypeTemplate is L2TaskBase {
             if (setRespectedGameTypes[chainId].l2ChainId != 0) {
                 DeputyGuardianModule dgm = DeputyGuardianModule(setRespectedGameTypes[chainId].deputyGuardian);
                 address portal = superchainAddrRegistry.getAddress(setRespectedGameTypes[chainId].portal, chainId);
-                dgm.setRespectedGameType(IOptimismPortal2(payable(portal)), setRespectedGameTypes[chainId].gameType);
+                dgm.setRespectedGameType(IAnchorStateRegistry(payable(portal)), setRespectedGameTypes[chainId].gameType);
```

Comment:
Would be helpful if someone with more context could take a look at this change.  

As is this fix doesn't make a lot of sense, but it does keep tests passing.  If I try to pull the actual ASR address here, there are some regression tests that break.